src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```diff
@@ -0,0 +1,42 @@
+.bleedWrapper {
+  background: var(--color-gray-300);
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: auto 0;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+  padding-bottom: 120px;
+  padding-top: 120px;
```

Comment:
The designs show 60px of padding above and below, not 120px. Also, there's supposed to be 50px of spacing on the left and right sides.

Reply:
👍 Changed padding top and bottom to 60px and padding left and right to 50px